hoover/search/models.py

```diff
@@ -126,6 +127,9 @@ def update(self):
     def user_access_list(self):
         return ', '.join(u.username for u in self.users.all())
 
+    def uploaders_access_list(self):
+        return ', '.join(u.username for u in self.uploaders.all())
```

Comment:
Return intersection between this and the `user_access_list`, since the admins may wrongly configure some user as an uploader but without having normal access. If that happens, the user shouldn't see or be able to upload anything.